### ISSUE TITLE
librdkafka 1.0.0

### ISSUE THREAD

**Issue by GeorgeShagov** [mfontanini/cppkafka#183]
Hello team!
Thank you for your great work!
The subjected release has a number or features like idempotent producer for instance that are highly desirable. I cannot tell though had they (Magnus and his team) made any changes into the interface or not, it might have happened they have not, but to make sure I am kindly asking you to confirm cppkafka release compatible with the latest librdkafka 1.0.0

Thx in advance
George

PS: you may add LATOKEN into the list of your clients. We do on cppKafka starting Dec, 2018

**Comment by accelerated** [comment]
Yes it's compatible since 1.0,0 remains API and ABI compatible with its previous version.
